### ISSUE TITLE
Suggestion: Allow upgrade directly in the right language

### ISSUE THREAD

**Issue by Spangenhelm** [Jermolene/TiddlyWiki5#1030]
Unfortunetly so far we can only do a direct upgrade in english (via the upgrade wizard), what about giving the possibility of upgrading directly in the right language (via a drop list or flags with every language available for example) ? Like we already do (thru "editions" of tw) but managed in the same place.

Basically, imo, every link about "downloading" tw should appear on the same page (but no matter if it is a new tw, a static one or an upgrade we could find all of them there) 
Maybe with a adress like this: get.tiddlywiki.com ? Short and simple.

Thanks

**Comment by Spangenhelm** [comment]
Hi @Jermolene any news about this ?
Thanks

**Comment by Jermolene** [comment]
I'm open to making `upgrade.html` translatable but I do think it's quite a lot of work. My preferred approach is to develop an in-place upgrade procedure so that any wiki can be upgraded directly, without having to visit upgrade.html.

The idea is to build on the recent work on the plugin library. We can construct an version of upgrade.html that can be embedded as an iframe.

**Comment by Spangenhelm** [comment]
Yes the auto-update idea sounds ideal !! You know my level in programming but if i can help then let me know.
